lametro/templates/lametro/index.html

```diff
@@ -141,17 +141,28 @@ <h2>
           serviceUrl: '{% url "autocomplete" %}',
           paramName: 'query',
           transformResult: function(response) {
-            var response = JSON.parse(response);
-            return {
-              suggestions: $.map(response.termHints, function(d) {
-                return {'value': d.name, 'data': d.id};
-              })
+            var res = JSON.parse(response);
+            if (res.status_code == 500 || res.termHints.length < 1) {
+              return {
+                suggestions: [{'value': 'No topics found', 'data': 'na'}]
```

Comment:
nice catch/update! teeny point of clarification: does this lib let you return an array of strings? if so, would be good to do that here, to avoid creating dummy data if we don't need to.